lms/static/js/discovery/views/course_card.js

```diff
@@ -33,10 +33,23 @@
                 var data = _.clone(this.model.attributes);
                 var userLanguage = '',
                     userTimezone = '';
+                var currentDate = new Date();
+                var courseStatus;
```

Comment:
Please initialize `courseStatus` variable with the string "The course is on" and then remove `else` statement at line 47
```suggestion
                var courseStatus = "The course is on";
```

Reply:
Done, thanks.